bullet_train/docs/super-scaffolding/dragula-sortable.md

```diff
@@ -0,0 +1,188 @@
+# Super Scaffolding with the `--sortable` option
```

Comment:
This is the old sortable docs, but updated with some info about how to continue to use `dragula` if you want to.